wp_localization_validation/src/ftl_file.rs

```diff
@@ -0,0 +1,86 @@
+use crate::TranslationLanguage;
+use std::{collections::HashMap, fmt::Debug, fs};
+use wp_localization_parser::TranslationEntry;
+
+/// The file extension for Fluent translation files.
+const FTL_EXTENSION: &str = "ftl";
+
+/// Parses all FTL files in the localization directory.
+///
+/// # Directory Structure
+///
+/// ```
+/// localization/
+///   en-US/
+///     main.ftl
+///   tr-TR/
+///     main.ftl
+/// ```
+///
+/// # Returns
+///
+/// A map of language codes to their translation entries.
+pub fn parse_localization_files(
+    localization_dir: &str,
+) -> Result<HashMap<TranslationLanguage, Vec<TranslationEntry>>, FtlSetupError> {
```

Comment:
Can we move this function to the parser crate? We could potentially reuse it in the wp_localization crate's build script.

Reply:
I think that'd be fine, but we can leave that to when we need it or when we are updating the build script especially since the PR is already merged.